spec/sass-spec.coffee

```diff
@@ -244,3 +244,39 @@ describe 'SCSS grammar', ->
       expect(tokens[1]).toEqual value: ':', scopes: ['source.css.scss', 'meta.set.variable.scss', 'punctuation.separator.key-value.scss']
       expect(tokens[2]).toEqual value: ' ', scopes: ['source.css.scss', 'meta.set.variable.scss']
       expect(tokens[3]).toEqual value: '$normal-font-size', scopes: ['source.css.scss', 'meta.set.variable.scss', 'variable.scss', 'variable.scss']
+
+  describe 'interpolation', ->
+    it 'is tokenized within single quotes', ->
+      {tokens} = grammar.tokenizeLine 'body { font-family: \'#\{$family}\'; }' # escaping CoffeeScript's interpolation
```

Comment:
To make this easier you can just switch the outside quotes to double-quotes, and then you won't have to escape the single quotes.
`"body { font-family: '#\{$family}'; }"`

Though why is the second bracket escaped?